TwitchPlaysAssembly/Src/TwitchPlaySettings.cs

```diff
@@ -42,6 +42,7 @@ public class TwitchPlaySettingsData
 	public bool ShowUnrecognizedCommandError = true;
 	public int ClaimCooldownTime = 30;
 	public int ModuleClaimLimit = 2;
+	public int FindClaimLimit = 3;
```

Comment:
This is a minor thing that I'll probably do but being able to set this to -1 to ignore the findclaim limit would be nice.